eth/stagedsync/stage_mining_finish.go

```diff
@@ -60,11 +60,13 @@ func SpawnMiningFinishStage(s *StageState, tx kv.RwTx, cfg MiningFinishCfg, quit
 		cfg.miningState.MiningResultPOSCh <- blockWithReceipts
 		return nil
 	}
+
+	// Note: To propose a new signer for Clique consensus, the block nonce should be set to 0xFFFFFFFFFFFFFFFF.
```

Comment:
this change affecting all consensuses, not only clique

Reply:
@AskAlexSharov already solved, pls check again.